Diary/Controller/DiaryViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  Diary - DiaryViewController.swift
+//  Created by rilla, songjun.
+//  Copyright © yagom. All rights reserved.
+// 
+
+import UIKit
+
+final class DiaryViewController: UIViewController {
+    private var diaries: [Diary]?
+    
+    private var dataSource: UITableViewDiffableDataSource<Section, Diary>!
+    
+    private lazy var tableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        return tableView
+    }()
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        decodeDiaryData()
+        configureUI()
+        configureLayout()
+        configureViewController()
+        configureTableView()
+    }
+    
+    private func decodeDiaryData() {
+        let decodedResult = DecodeManager().decodeJSON(fileName: "sample", type: [Diary].self)
+        diaries = try? verifyResult(result: decodedResult)
+    }
+    
+    private func verifyResult<T, E: Error>(result: Result<T, E>) throws -> T? {
+        switch result {
+        case .success(let data):
+            return data
+        case .failure(let error):
+            throw error
+        }
+    }
+    
+    private func configureUI() {
+        view.addSubview(tableView)
+    }
+    
+    private func configureLayout() {
+        let safeArea = view.safeAreaLayoutGuide
+        
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(equalTo: safeArea.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: safeArea.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: safeArea.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: safeArea.bottomAnchor)
+        ])
+    }
+    
+    private func configureViewController() {
+        view.backgroundColor = .white
+        self.title = "일기장"
+        
+        let buttonItem: UIBarButtonItem = {
+            let button = UIButton()
+            
+            button.setImage(UIImage(systemName: "plus"), for: .normal)
+            button.addTarget(self,
+                             action: #selector(presentAddingDiaryPage),
+                             for: .touchUpInside)
```

Comment:
```suggestion
            button.addTarget(
                self,
                action: #selector(presentAddingDiaryPage),
                for: .touchUpInside
            )
```

저는 개인적으로 이렇게 줄바꿈 하는 것을 선호하는데, 요건 스타일의 차이니까 참고만 해주세요~